### DISCUSSION TITLE
Containerlab support for IOL

### DISCUSSION THREAD

**Discussion by DanPartelly** [ipspace/netlab#1342] [General]
It seems that Containerlab added support for IOL binaries in this commit. So networklab can get it almost free as well.

https://github.com/srl-labs/containerlab/commit/2764acc763087740655cf440dd1d348a91c93548

I also asked Roman about fd.io VPP. He is open to onboard it, if we/I can get it to work into a container.

**Comment by ipspace** [comment]
Nice. I would still wait to have it included in a containerlab release

Of course we can support for it. Want to do it and submit a PR or shall we use the other trick we used in the past? ;)

As for VPP, go for it ;))

**Reply by DanPartelly** [reply]
Yeah sure. I can try my had at it in weekend.  Never wrote one line of python, so this should be funny. You'll get the SSH timeout patch soon as well, I still owe you that.

**Comment by ipspace** [comment]
Implemented / tested.
